config/routes.rb

```diff
@@ -0,0 +1,64 @@
+Rails.application.routes.draw do
+
+root to: 'posts#index'
+
+resources :posts
+resources :comments
```

Comment:
Based on how your routes and controller are currently set up, how would you go about accessing a comment(s) that belongs to a particular post?